app/form/Mappings.scala

```diff
@@ -48,10 +48,12 @@ object Mappings extends DateMappings {
 
   val addressMapping: Mapping[Address] = mapping(
     "addressId" -> addressId,
-    "line1"     -> onlyIf(addressEnteredManually, text.verifying(nonEmpty, maxLength(36)))(""),
-    "line2"     -> text(maxLength = 36),
-    "line3"     -> text(maxLength = 36),
-    "line4"     -> text(maxLength = 36),
+    "line1" -> onlyIf(
+      addressEnteredManually,
+      text.verifying(nonEmpty, maxLength(30, "enrolment.address.maxLengthError")))(""),
```

Comment:
I think line one always has a limit of 80 characters according to Mahmed's comment on [VTCCA-6394](https://jira.tools.tax.service.gov.uk/browse/VTCCA-6394)

Reply:
okay i'll update the first line to be 80, and the corresponding error message too